### ISSUE TITLE
[VS Code] Error: Running the contributed command:'extension.git-file-history' failed.

### ISSUE THREAD

**Issue by MatiasOlivera** [pomber/git-history#97]
Hi folks, I just installed the extension and when trying to see the history of a file I get the following error:

```
notificationsAlerts.ts:38 Error: Running the contributed command:'extension.git-file-history' failed.
    at e._executeContributedCommand (c:\Users\MyUser\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:486:51)
    at e.$executeContributedCommand (c:\Users\MyUser\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:486:443)
    at t._doInvokeHandler (c:\Users\MyUser\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:644:118)
    at t._invokeHandler (c:\Users\MyUser\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:643:744)
    at t._receiveRequest (c:\Users\MyUser\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:642:322)
    at t._receiveOneMessage (c:\Users\MyUser\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:641:155)
    at c:\Users\MyUser\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:638:960
    at c:\Users\MyUser\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:38:994
    at e.fire (c:\Users\MyUser\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:44:113)
    at a (c:\Users\MyUser\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:170:213)
    at Socket.f._socketDataListener (c:\Users\MyUser\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:170:434)
    at Socket.emit (events.js:182:13)
    at Socket.EventEmitter.emit (domain.js:442:20)
    at addChunk (_stream_readable.js:279:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at Pipe.onread (net.js:636:20)
t.onDidNotificationChange @ notificationsAlerts.ts:38
(anonymous) @ notificationsAlerts.ts:26
e.fire @ event.ts:567
t.notify @ notifications.ts:113
t.error @ notificationService.ts:48
t.onError @ commandsHandler.ts:325
(anonymous) @ commandsHandler.ts:310
_tickCallback @ internal/process/next_tick.js:68
Promise.then (async)
(anonymous) @ commandsHandler.ts:306
setTimeout (async)
t.runAction @ commandsHandler.ts:296
t.run @ commandsHandler.ts:280
e.run @ quickOpenModel.ts:583
t.elementSelected @ quickOpenWidget.ts:556
(anonymous) @ quickOpenWidget.ts:235

```

**Comment by pomber** [comment]
Hey Matias, try with the new version (v0.1.0)

**Comment by MatiasOlivera** [comment]
@pomber I tried with the new version but still doesn't work.

It shows the message: 
`Loading c:pathtoclientsrcApp.vue history ...`

The path should be `c:\path\to\client\src\App.vue`

**Comment by pomber** [comment]
Sorry about that, and thanks for reporting it again. Just published 0.1.1 and tested it on windows. It should work now.

**Comment by MatiasOlivera** [comment]
It is working now. Thank you! 🎉
